packages/commonwealth/client/scripts/views/components/ProposalSelector/ProposalSelector.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import 'components/ChainEntitiesSelector.scss';
```

Comment:
you changed tsx file names, but scss files are unchanged. We should also change it to ProposalSelector.scss, along with the css class below in line 73